webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/nodes/Properties.scala

```diff
@@ -446,6 +471,33 @@ object GenericProperty {
       case JArray(arr)      => ConfigValueFactory.fromIterable(arr.map(x => fromJsonValue(x)).asJava)
     }
   }
+  def fromZioJson(value: zio.json.ast.Json): ConfigValue = {
```

Comment:
`zio-json` AST, used to parse a generic config value (that can be any json) into a property value.